webapi/CopilotChat/Models/ChatSession.cs

```diff
@@ -29,6 +29,12 @@ public class ChatSession : IStorageEntity
     [JsonPropertyName("createdOn")]
     public DateTimeOffset CreatedOn { get; set; }
 
+    /// <summary>
+    /// Initial bot message.
+    /// </summary>
+    [JsonPropertyName("initialBotMessage")]
+    public ChatMessage? InitialBotMessage { get; set; }
```

Comment:
What is the purpose of saving the initial message in the session? This couples the ChatMessage and ChatSession models.

Reply:
It helps optimize our network calls from the webapp (why load the messages when we already know what it is)? What's the reservation with coupling the two models?

Reply:
Optionally, I can create a new model called CreateChatResponse that's not coupled to a storage repository

Reply:
I think it's better to not save an entry of the ChatMessage model in the ChatSession model. This will have three disadvantages with the only benefit of reducing one single network call:
1. It essentially creates two copies of the initial message in the database, one in the session repository, the other in the message repository.
2. It breaks the pattern of loading/handling sessions and messages separately. 
3. It limits our ability to create multiple initial messages (not likely to happen but imagine if we want an introduction message then a greeting message).